addonscript/tox.ini

```diff
@@ -34,7 +34,7 @@ depends =
 [testenv:report]
 skip_install = true
 commands = coverage report -m
-depends = py38
+depends = py39
```

Comment:
We probably want to also update the top level [tox.ini](https://github.com/mozilla-releng/scriptworker-scripts/blob/master/tox.ini#L51) to move the `check` target to the py39 (or whatever is the shipping python minor version) per script as well.